### DISCUSSION TITLE
Can 'C:\Users\XXX\AppData\Local\Packages' be changed?

### DISCUSSION THREAD

**Discussion by sharpninja** [microsoft/winget-cli#1378] [Q&A]
I can no longer run executables from `AppData` and children due to CarbonBlack and have a dedicated path I am allowed to execute code from.  I need to relocate this location, is there a way to do that?

**Comment by denelon** [comment]
I think you might be referring to https://github.com/microsoft/winget-cli/issues/1213. If that's the case please add a 👍

**Reply by sharpninja** [reply]
Can it be a persisted setting?

**Reply by denelon** [reply]
The intent would be both a command-line argument, and a setting.
